PLAYER/DisablePlayerVehicleRewards.md

```diff
@@ -7,7 +7,9 @@ ns: PLAYER
 // 0xC142BE3BB9CE125F 0x8C6E611D
 void DISABLE_PLAYER_VEHICLE_REWARDS(Player player);
 ```
-
+```
```

Comment:
Code blocks are meant for NativeDB imports

Reply:
Was in the process of removing them. Is this better? (My first time contributing to the docs, sorry!)